docs/ipfs-cluster-guide.md

```diff
@@ -121,9 +121,21 @@ Each section of the configuration file and the options in it depend on their ass
       "proxy_read_timeout": "10m0s",                          // Here and below, timeouts for network operations
       "proxy_read_header_timeout": "5s",
       "proxy_write_timeout": "10m0s",
-      "proxy_idle_timeout": "1m0s"
+      "proxy_idle_timeout": "1m0s",
+      "pin_method": "pin"                                     // Supports "pin" and "refs". "refs" will fetch all deps before pinning.
+                                                              // Use refs when GC is disabled on ipfs.
+                                                              // Increase maptracker.concurrent_pins to take advantange of concurrency.
     }
   },
+  "pin_tracker": {
+    "maptracker": {
+      "pinning_timeout": "1h0m0s",                            // How long before we transition a pinning CID to error state
```

Comment:
Sometimes it takes 10 days to add a pin. Right now this sets PIN_ERROR on the pins that have been going on for too long when a `sync` happens. The idea with an improved pintracker is that this would also cancel the whole request to ipfs. Maybe something that is not provided by anyone is being pinned. Right now is pretty useless as it doesn't remove queued items and if something else is pinning everything in the queue will be set to pin error.

Reply:
10 days :scream_cat:

Reply:
That's what happens when your seeder provides gibabyte pins at 50KB/s